docs/platform-packet.md

```diff
@@ -34,6 +34,11 @@ an additional YAML for [arm64](../examples/packet.arm64.yml) servers
 which provide both access to the serial console and via ssh and
 configures bonding for network devices via metadata (if supported).
 
+For x86_64 builds for Intel servers we strongly recommend adding
+`ucode: intel-ucode.cpio` to the kernel section in the YAML. This
+updates the Intel CPU microcode to the latest by prepending it to the
+generated initrd file.
```

Comment:
Should the word "baremetal" appear in here somewhere?

Reply:
added more text to make it clear that it is recommended for baremetal and shouldn't be used for VM images (but it's harmless there)